package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Extension Activity Monitor provides more transparency which could help to increase the reliability of abuse reporting and accountability on developers. It is an additional useful tool to aid investigating bugs in the extensions or in the Firefox WebExtensions internals.",
   "scripts": {
+    "start:firefox": "web-ext run -s src -f nightly --pref=\"extensions.experiments.enabled=true\"",
```

Comment:
@atiqueahmedziad if we do add a "webExt" property to this package.json file, as I did mention in our meeting earlier today ([also documented here in this doc page related to the web-ext cli tool](https://extensionworkshop.com/documentation/develop/getting-started-with-web-ext/#setting-option-defaults-in-a-configuration-file)), then we should be able to:
- reduce this command to the bare minimum (`"start:firefox": "web-ext run"`)
- reduce the comment suggested if the user want to pass other options to just the options that would differ (e.g. `web-ext run --firefox-binary /path/to/your/nightly/firefox`), because  all the other parameters would default to the ones specified in the configuration included in the package.json file.

The configuration to be included in the package.json should look more or less like:
```json
  "devDependencies": {
    ...
  },
  "webExt": {
    "sourceDir": "src",
    "run": {
      "pref": ["extensions.experiments.enabled=true"],
      "firefox": "nightly"
    }
  }
}
```


Given that this extension isn't going to run on anything but Firefox Desktop (at least at the moment), we may call the script just "start" instead of "start:firefox".

Reply:
Sorry, I should have asked about it since I was confused with setting up `pref`. Thanks for explaining. Added it.